src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/AsciidocToHtml.kt

```diff
@@ -0,0 +1,55 @@
+package nl.avisi.structurizr.site.generatr.site.model
+
+import org.asciidoctor.Asciidoctor
+import org.asciidoctor.Options
+import org.asciidoctor.SafeMode
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import nl.avisi.structurizr.site.generatr.site.asUrlToFile
+
+fun asciidocToHtml(pageViewModel: PageViewModel, asciidoc: String, svgFactory: (key: String, url: String) -> String?): String {
+    val asciidoctor = Asciidoctor.Factory.create()
+    val options = Options.builder()
+            .safe(SafeMode.SERVER)
```

Comment:
Could you please align the indentation here (and in other places) with the rest of the project?